### PR TITLE
Fix gosec 107 - request with variable URL

### PR DESCRIPTION
**Pull request by pavolloffay**
Fixes failing PRs builds.

Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

**Comment by yurishkuro** [comment]
@pavolloffay could you explain in the description what/why is this solving?

File: crossdock/services/agent.go [hunk 1]
```diff
@@ -54,8 +54,7 @@ func getSamplingURL(url string) string {
 
 // GetSamplingRate returns the sampling rate for the service-operation from the agent service.
 func (s *agentService) GetSamplingRate(service, operation string) (float64, error) {
-	url := fmt.Sprintf(getSamplingURL(s.url), getTracerServiceName(service))
-	resp, err := http.Get(url)
+	resp, err := http.Get(fmt.Sprintf(getSamplingURL(s.url), getTracerServiceName(service)))
```
[the remark this answers is not in the document]
Comment (yurishkuro) [reply crossdock/services/agent.go]: what is it complaining about, specifically?

**Comment by yurishkuro** [comment]
I see these errors in my PR:

> [/home/travis/gopath/src/github.com/jaegertracing/jaeger/plugin/storage/es/spanstore/writer.go:182] - G107: Potential HTTP request made with variable url (Confidence: MEDIUM, Severity: MEDIUM)

I think this is utter nonsense. Either we disable this specific G107 role, or throw away gosec.

**Comment by pavolloffay** [comment]
+1 Let's disable it

File: crossdock/services/agent.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (yurishkuro) [reply crossdock/services/agent.go]: let's use `// #nosec G107`, not hack the code.

**Pull request merged**